src/components/DocumentationTopic.vue

```diff
@@ -302,6 +297,17 @@ export default {
     shouldShowLanguageSwitcher: ({ objcPath, swiftPath }) => objcPath && swiftPath,
     hideSummary: () => getSetting(['features', 'docs', 'summary', 'hide'], false),
     enhanceBackground: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
+    technologies({ modules, technology }) {
+      if (!modules) return [];
```

Comment:
```suggestion
```

I don't think this is needed since calling `modules.reduce` when `modules` is empty should already return `[]`.